### PR TITLE
prototyping RV profiles

### PR DESCRIPTION
**Pull request by nibrunie**
I would like to add support for profile parsing, but I am not sure how to do it

### CODE REVIEW

File: lib/exto.js [hunk 1]
```diff
@@ -225,6 +225,21 @@ exto.zvks = {
...
 };
 
+// RISC-V profiles
+exto.rvi20u32 = {
+  desc: 'RVI20 32-bit unpriviledged profile',
+  items: [exto.i]
+};
+
+exto.rvi20u64 = {
+  desc: 'RVI20 32-bit unpriviledged profile',
```
Comment (nibrunie) [inline lib/exto.js]:
```suggestion
  desc: 'RVI20 64-bit unprivileged profile',
```

File: lib/exto.js [hunk 2]
```diff
@@ -225,6 +225,21 @@ exto.zvks = {
   items: [exto.zvkb, exto.zvksh, exto.zvksed]
 };
 
+// RISC-V profiles
+exto.rvi20u32 = {
+  desc: 'RVI20 32-bit unpriviledged profile',
```
Comment (nibrunie) [inline lib/exto.js]:
```suggestion
  desc: 'RVI20 32-bit unprivileged profile',
```

File: lib/exto.js [hunk 3]
```diff
@@ -225,6 +225,21 @@ exto.zvks = {
...
+  items: [exto.i]
+};
+
+exto.rvi20u64 = {
+  desc: 'RVI20 32-bit unpriviledged profile',
+  items: [exto.i]
+};
+
+exto.rva20u64 = {
+  desc: 'RVA20user-mode profile for 64-bit application processors',
```
Comment (nibrunie) [inline lib/exto.js]:
```suggestion
  desc: 'RVA20 user-mode profile for 64-bit application processors',
```

**Comment by drom** [comment]
good first step towards #11

**Comment by drom** [comment]
Let me merge your part and fix the parser.

**Pull request merged**